app/controllers/Application.scala

```diff
@@ -186,4 +188,29 @@ final class Application @Inject()(data: DataHelper,
     Redirect(ShowHome)
   }
 
+  /**
+    * Show stats
+    * @return
+    */
+  def showStats() = (Authenticated andThen PermissionAction[AuthRequest](ViewStats)) { implicit request =>
+    val queryReviews: Future[Seq[(String,String)]] = this.service.DB.db.run(sql"SELECT foo.data, CAST(foo.when AS DATE) FROM (WITH backup AS (SELECT 0,current_date - (INTERVAL '1 day' * generate_series(0, 9)) days) SELECT COALESCE(data.count, 0) AS data, COALESCE(data.when, backup.days) AS when FROM (SELECT COUNT(*), CAST(approved_at AS DATE) AS when FROM project_versions WHERE approved_at IS NOT NULL GROUP BY CAST(approved_at AS DATE)) data FULL OUTER JOIN backup ON data.when = backup.days) AS foo".as[(String, String)])
```

Comment:
This looks ugly and unmaintainable. You could make one template SQL file and modify it for the different queries.

Reply:
maybe we should make it a sql view we that would be more suitable in this situation
what do you think is the best way?

Reply:
at the very least, break up the query into multiple lines

Reply:
A twirl template might be overcomplicating things. A file with some placeholders would do fine.